src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -907,6 +916,16 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
                enable_apiserver_vnet_integration=False,
                apiserver_subnet_id=None,
                ):
+
+    if disable_disk_driver or disable_file_driver:
+        msg = 'Did you check if the related PVs and PVCs were deleted before disabling the CSI Drivers?'
+        if not yes and not prompt_y_n(msg, default="n"):
+            return None
+    if disable_snapshot_controller:
+        msg = 'Did you check if VolumeSnapshots, VolumeSnapshotClasses and VolumeSnapshotContents CRDs and their resources were deleted before disabling SnapshotController?'
```

Comment:
`please make sure there are no existing Kubernetes resources that are used by the snapshot controller`

Reply:
@olsenme are you ok with the above statement in the cli command?

Reply:
Changed.